MSAL/test/unit/MSALPublicClientApplicationTests.m

```diff
@@ -3060,6 +3060,49 @@ - (void)testSignoutWithAccount_whenNonNilaccount_andSignoutFromBrowserFalse_andB
     [self waitForExpectations:@[expectation] timeout:1];
 }
 
+- (void)testSignoutWithAccount_whenSignoutFromBrowserTrue_andNilWebViewParameters_shouldFailWithError_andNotRemoveAccount
```

Comment:
Just curious, should we include a test case for _whenSignoutFromBrowserFalse_andNilWebViewParameters_shouldremoveAccount?

Reply:
Great idea, added :)